src/rebuild/ras.c

```diff
@@ -0,0 +1,60 @@
+/**
+ * (C) Copyright 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * rebuild: RAS
+ *
+ * This file contains RAS helpers for rebuild events.
+ */
+#define D_LOGFAC       DD_FAC(rebuild)
+
+#include <daos_srv/daos_server.h>
+
+static int
+raise_ras(ras_event_t id, uuid_t *pool, char *op_str, char *msg)
+{
+	if ((pool == NULL) || uuid_is_null(*pool)) {
+		D_ERROR("invalid pool\n");
+		return -DER_INVAL;
+	}
+
+	ds_notify_ras_event(id, msg, RAS_TYPE_INFO, RAS_SEV_INFO,
+			    NULL /* hwid */, NULL /* rank */,
+			    NULL /* jobid */, pool, NULL /* cont */,
+			    NULL /* objid */, NULL /* ctlop */, op_str);
+	return 0;
+}
+
+int
+rebuild_notify_ras_start(uuid_t *pool, char *op_str)
```

Comment:
naming would be more intuitive as "notify_ras_rebuild_start" IMO

Reply:
As it's private to the rebuild module I'm following the convention set by the rest of the codebase.